### PR TITLE
👌 smaller format

### PR DESCRIPTION
**Pull request by MJEND7**
I simply changed all your old functions to arrow functions. smaller 👌

### CODE REVIEW

File: lib/conversions.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+'use strict';
```
Comment (u9g) [inline lib/conversions.js:1]: not needed, we lint with standardjs instead

**Comment by Karang** [comment]
Tests are not passing, lint with `npm run fix`

File: lib/conversions.js (hunk 1 above)
Comment (KaffinPX) [reply lib/conversions.js:1]: yeah

Comment (MJEND7) [reply lib/conversions.js:1]: ok

**Comment by MJEND7** [comment]
O I see my problem

**Pull request closed without merging**